ltsp/common/ltsp/55-images.sh

```diff
@@ -327,6 +327,24 @@ mount_file() {
     die "I don't know how to mount $src"
 }
 
+# Mount with squashfuse if /dev/loop-control is missing
+mount_squashfuse_or_loop() {
+    local src dst options
+
+    src=$1
+    dst=$2
+    # Shift src and dst coordinates
```

Comment:
"# Shift src and dst coordinates" -- probably in the category of meaningless/misleading comment. ("misleading" only in that does anyone really use the term "coordinates"?)